File: templates/welcome/Docs (chosenLanguage === 'JavaScript')/hello.js
<%
const kebabCaseName = name
    .trim()
    .toLocaleLowerCase()
    .split(' ')
    .join('-');
const capitalizedName = capitalize(
    dashToCamel(kebabCaseName)
);

%>class Hello<%- capitalizedName %> {

}

const welcomed<%- capitalizedName %> = new Hello<%- capitalizedName %>()

File: templates/welcome/Docs (chosenLanguage === 'ruby')/hello.rb
<%
const capitalizedName = capitalize(
    dashToCamel(
        name
            .split(' ')
            .join('-')
        )
)

%>class Welcome<%- capitalizedName %>
end

welcomed_<%- name.split(' ').join('_').toLocaleLowerCase() %> = Welcome<%- capitalizedName %>. new